### ISSUE TITLE
training parameters

### ISSUE THREAD

**Issue by eksuas** [8uos/HashGAN-Pytorch#2]
Hello :)

Thanks for the study! 
Could you please share your training parameters such as batch-size, the number epochs, learning rate etc.
I could not obtain the similar results as the paper on the both datasets with default parameters.
Thank you!

**Comment by 8uos** [comment]
Hi, 
I also failed to reproduce the reported values in the paper.
Please see #1, this is a previous issue about that problem.

**Comment by eksuas** [comment]
> Hi,
> I also failed to reproduce the reported values in the paper.
> Please see #1, this is a previous issue about that problem.

Thanks for quick response!
